app/models/facility.rb

```diff
@@ -243,15 +218,12 @@ def teleconsultation_enabled?
   end
 
   def teleconsultation_phone_number_with_isd
-    teleconsultation_phone_number = teleconsultation_phone_numbers.first
-    return if teleconsultation_phone_number.blank?
-
-    Phonelib.parse(teleconsultation_phone_number.isd_code + teleconsultation_phone_number.phone_number).full_e164
+    teleconsultation_phone_numbers_with_isd.first&.dig(:phone_number)
   end
 
   def teleconsultation_phone_numbers_with_isd
-    teleconsultation_phone_numbers.map do |phone_number|
-      {phone_number: Phonelib.parse(phone_number.isd_code + phone_number.phone_number).full_e164}
+    teleconsultation_medical_officers.map do |medical_officer|
+      {phone_number: medical_officer.full_teleconsultation_phone_number}
```

Comment:
Grab the MO's numbers instead of the facility attribute. This method powers the `api/v4/facility_teleconsultations` endpoint that may still be in use.

Reply:
Hrm, didn't realize this method was sticking this data in a hash. Can't this just return the list and the hash stuff can happen at the call site? This hash seems a little arbitrary in this model.

Ignore if too much change.

Reply:
If the touchpoints are few, I'll do it right away. If they are numerous, I'll leave it alone.